FreeRTOS/FreeRTOS-Plus-TCP
# eARPGetCacheEntryGateWay() return value with no valid gateway

Description
-----------
When an IPv4 packet needs to be sent to a host outside the local network, we need to send this packet to the gateway.
If there is no valid gateway, we cannot send the packet.

Test Steps
-----------
1. Configure an single IPv4 endpoint with and IP of 192.168.0.1/24 and a gateway of 0.0.0.0
2. Open a TCP connection to 10.10.10.10
3. `eARPGetCacheEntryGateWay()` should result in `eCantSendPacket`


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
